### DISCUSSION TITLE
Decreasing constraint length

### DISCUSSION THREAD

**Discussion by DarXector** [enable3d/enable3d#291] [Help / Questions]
Hi, 

I am trying to connect several meshes with constraints and then gradually decrease the distance of those objects by shortening those constraints. For example having a crane that pulls a box up. I have tried using PointToPoint and Spring constraints but I cannot figure out how to decrease the constraint length.

Is there way to do it or using constraints is an overkill?

Thank you!

**Comment by yandeu** [comment]
Maybe a [slider constraint with a motor](https://knowledge.autodesk.com/support/maya/learn-explore/caas/CloudHelp/cloudhelp/2019/ENU/Maya-SimulationEffects/files/GUID-CDB3638D-23AF-49EF-8EF6-53081EE4D39D-htm.html)?

_But I'm not sure if this is available in ammo.js_
